ProjectShop/OrderItems/admin.py

```diff
@@ -0,0 +1,5 @@
+from django.contrib import admin
+from .models import OrderItems
```

Comment:
Please, add an absolute path to all imports:
from OrderItems.models import OrderItems